src/gs-overview-page.c

```diff
@@ -486,8 +486,20 @@ get_categories_cb (GObject *source_object,
 	gs_shell_side_filter_clear_categories (priv->shell);
 
 	for (guint i = 0; i < list->len; i++) {
+		gboolean usb_category_inserted = FALSE;
```

Comment:
Commenting here because there's no way of commenting in the commit message: Please capitalize the first word (display -> Display). This is not done when the first word is the plugin/module's name of course.

Reply:
Fixed locally